### PR TITLE
Fix categories 

### PR DESCRIPTION
**Pull request by saulakravitz**
Address [NVSSIG-72](https://jira.mitre.org/browse/NVSSIG-72)
- ensure that for each group, the same LOINC code is used for all Observations (code), and all Procedures(category), and all Conditions (category[requiredCategory]).

This has been built on this branch with no apparent new issues.

### CODE REVIEW

File: input/fsh/profiles/ObservationNICUAdmission.fsh [hunk 1]
```diff
@@ -9,4 +9,4 @@ Description: "This profile indicates that the abnormal condition of an admission
 * subject only Reference(PatientChildVitalRecords)
 * value[x] 1..
 * value[x] only CodeableConcept
-* value[x] = $sct#830077005 "Admission to neonatal intensive care unit (procedure)"
```
Comment (ptran265) [inline input/fsh/profiles/ObservationNICUAdmission.fsh:12]: won't this make display required?

**Review by ptran265 — APPROVED** [review]
looks good besides that one change where the display is required for NICU admission

File: input/fsh/instances/condition-infection-present-during-pregnancy-jada-ann-quinn.fsh [hunk 2]
```diff
@@ -4,7 +4,7 @@ Title: "Condition - Infection Present During Pregnancy - Jada Ann Quinn"
 Description: "Condition - Infection Present During Pregnancy: Jada Ann Quinn example"
 Usage: #example
 * category[us-core] = $condition-category#problem-list-item "Problem List Item"
-* category[infectionDuringPregnancy] = $loinc#72519-2 "Infections present AndOr treated during this pregnancy for live birth [US Standard Certificate of Live Birth]"
+* category[requiredCategory] = $loinc#72519-2 "Infections present AndOr treated during this pregnancy for live birth [US Standard Certificate of Live Birth]"
```
Comment (rpassas) [inline input/fsh/instances/condition-infection-present-during-pregnancy-jada-ann-quinn.fsh]: I don't think we need to have instances include the category explicitly. Other conditions do not include it (e.g., Chorioamnionitis).

Comment (saulakravitz) [reply input/fsh/instances/condition-infection-present-during-pregnancy-jada-ann-quinn.fsh]: Correct.

File: input/fsh/profiles/ObservationNICUAdmission.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/profiles/ObservationNICUAdmission.fsh:12]: YEs... will fix...

**Pull request merged**